pkg/output/stdout.go

```diff
@@ -155,13 +155,28 @@ func (s *Stdoutput) resultQuiet(resp ffuf.Response) {
 	}
 }
 
+func (s *Stdoutput) getRedirectLocation(resp *ffuf.Response) string {
```

Comment:
I'm thinking that a public method in [ffuf.Response](https://github.com/ffuf/ffuf/blob/master/pkg/ffuf/response.go) struct might be the best place for this, as then the code can be reused throughout the codebase.

Reply:
ow Okay, did not see this one !